playbooks/roles/edxapp/templates/edx/bin/edxapp-update-assets.j2

```diff
@@ -4,10 +4,11 @@
 {% for override in edxapp_staticfiles_storage_overrides %}
 sudo -E -H -u {{ edxapp_user }} \
     env "PATH=$PATH" "STATICFILES_STORAGE={{ override }}" \
-    {{ edxapp_venv_bin }}/paver update_assets --debug-collect --settings=$EDX_PLATFORM_SETTINGS
+    {{ edxapp_venv_bin }}/npm run build && {{ edxapp_venv_bin }}/manage.py lms collectstatic --noinput --debug-collect --settings=$EDX_PLATFORM_SETTINGS && {{ edxapp_venv_bin }}/manage.py cms collectstatic --noinput --debug-collect --settings=$EDX_PLATFORM_SETTINGS
```

Comment:
Shouldn't it be this, because `manage.py` won't be in `edxapp_venv_bin`, or will it be there?
```suggestion
    {{ edxapp_venv_bin }}/npm run build && {{ edxapp_venv_bin }}/python manage.py lms collectstatic --noinput --debug-collect --settings=$EDX_PLATFORM_SETTINGS && {{ edxapp_venv_bin }}/python manage.py cms collectstatic --noinput --debug-collect --settings=$EDX_PLATFORM_SETTINGS
```